src/components/course/ToggleOptIn.vue

```diff
@@ -1,8 +1,9 @@
+<!-- ToggleOptIn.vue -->
```

Comment:
@mohamalnadi - Can we revert all changes to `ToggleOptIn` since you've removed its usage from other components in this PR?  (I have working changes to `ToggleOptIn` and the changes here would cause conflicts in my work.)